### ISSUE TITLE
Suggest a valid 12th or 24th word.

### ISSUE THREAD

**Issue by GeetaCX** [KeystoneHQ/keystone3-firmware#928]
There are many custom entropy technique that produce 11 or 23 words and requires the wallet to generate a valid checksum word.

**Comment by soralit** [comment]
Hi @GeetaCX , thanks for your suggestion. Many users request this feature these days so we will plan it into our development plan. Just a confirmation, do you want this function to auto fill the last word when you import a seed phrase to Keystone 3 Pro?

**Comment by GeetaCX** [comment]
Yes, that is a very important feature to me, there should only be a handful of possible last words.
